pkg/executor/insert.go

```diff
@@ -193,7 +197,16 @@ func (e *InsertValues) prefetchDataCache(ctx context.Context, txn kv.Transaction
 }
 
 // updateDupRow updates a duplicate row to a new row.
-func (e *InsertExec) updateDupRow(ctx context.Context, idxInBatch int, txn kv.Transaction, row toBeCheckedRow, handle kv.Handle, _ []*expression.Assignment, dupKeyCheck table.DupKeyCheckMode, autoColIdx int) error {
+func (e *InsertExec) updateDupRow(
```

Comment:
BTW, there are a lot of the format changes (e.g. function signature, error handling). Are they reported by linter, or just formated by your IDE/editors?

If it'll not break the CI, I'd suggest not to change them. It'll make this patch cleaner.

Reply:
Not by linter actually. I used to change format of related code in the PR, to make the function signature clearer.